seed/seed.json

```diff
@@ -134,6 +134,122 @@
                 "channel": ["NIGHTLY", "BETA"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "AdNotificationsDisabledOnBetaAndReleaseChannelsStudy",
+            "experiments": [
+                {
+                    "name": "AdNotifications",
+                    "probability_weight": 100,
+                    "parameters": [
+                        {
+                            "name": "should_show_custom_notifications",
+                            "value": "false"
+                        }
+                    ],
+                    "feature_association": {
+                        "enable_feature": ["AdNotifications"],
+                        "disable_feature": []
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "channel": ["BETA", "RELEASE"],
+                "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
+            }
+        },
+        {
+            "name": "BraveAds.AdNotificationsEnabledOnUnknownAndNightlyChannelStudy",
+            "experiments": [
+                {
+                    "name": "AdNotifications",
+                    "probability_weight": 40,
+                    "parameters": [
+                        {
+                            "name": "should_show_custom_notifications",
+                            "value": "false"
+                        }
+                    ],
+                    "feature_association": {
+                        "enable_feature": ["AdNotifications"],
+                        "disable_feature": []
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 60
```

Comment:
I see `AdNotifications` is [enabled by default](https://github.com/brave/brave-core/blob/6cda51428c6a1a8268d3dcdf4485ca9cdb9601af/components/brave_ads/browser/features.cc#L13-L14). This is dangerous in general for new features, and also this PR doesn't do what you want it to do: it doesn't disable the feature for 60%.

You either need to explicitly disable the feature here, or disable it in code and enable it using variations (preferred way).
Explicit disable for default-enabled feature is not the best solution, because variations applied only after restart, it means that the first run a feature will always be enabled.

Reply:
my understanding is that this is an old feature that just received a feature flag, so somewhat unusual as we're just introducing features to ads. In this case the group split should be controlled by `should_show_custom_notifications`, where old/default behaviour is "should not show" and the configuration under test is "should show". Does that make sense?

Reply:
the study should probably be called `BraveAds.CustomAdNotificationsEnabledOnUnknownAndNightlyChannelStudy`, wdyt @tmancey?

Reply:
Agreed. Thanks

Reply:
Fixed

Reply:
> my understanding is that this is an old feature that just received a feature flag, so somewhat unusual as we're just introducing features to ads. In this case the group split should be controlled by should_show_custom_notifications, where old/default behaviour is "should not show" and the configuration under test is "should show". Does that make sense?

yep, it makes sense now, but very unusual indeed. thanks!